### PR TITLE
Update _fakenameof.aiml

### PR DESCRIPTION
**Pull request by zinnerzPT**
Added 41 new fortunes and fixed typos of old ones

### CODE REVIEW

**Comment by JustArchi** [comment]
Initial :+1: , waiting for owner review.

File: AIML/lib/aiml/_fakenameof.aiml [hunk 1]
```diff
@@ -299,7 +298,7 @@
 <li>Shoot for the moon! If you miss you will still be among the stars.</li>
 <li>A pleasant surprise is waiting for you.</li>
 <li>A nasty surprise is waiting for you.</li>
-<li>A virtual forune cookie will not satisfy your hunder likes that of a homemade one.</li>
+<li>A virtual fotrune cookie will not satisfy your hunger likes that of a homemade one.</li>
```
Comment (JustArchi) [inline AIML/lib/aiml/_fakenameof.aiml]: Fix typo.

Comment (zinnerzPT) [inline AIML/lib/aiml/_fakenameof.aiml]: Fixed.

File: AIML/lib/aiml/_fakenameof.aiml [hunk 2]
```diff
@@ -309,6 +308,48 @@
 <li>That'll be $0.99.</li>
 <li>:3</li>
 <li>Help! I am trapped in a fortune cookie machine!</li>
+<li>A friend asks only for your time not your money.</li>
```
Comment (FakeNameOF) [inline AIML/lib/aiml/_fakenameof.aiml]: Potential duplicate

File: AIML/lib/aiml/_fakenameof.aiml [hunk 3]
```diff
@@ -309,6 +308,48 @@
...
+<li>You can make your own happiness.</li>
+<li>It's better to be alone sometimes.</li>
+<li>When fear hurts you, conquer it and defeat it!</li>
+<li>The man on the top of the mountain did not fall there.</li>
+<li>Stop wishing. Start doing.</li>
+<li>DEDICATION MOTIVATION ACCELERATION</li>
+<li>Stay true to those who would do the same for you.</li>
+<li>Ask yourself if what you are doing today is getting you closer to where you want to be tomorrow.</li>
+<li>Your fortune is as sweet as a cookie.</li>
+<li>When hungry, order more Chinese food.</li>
```
Comment (FakeNameOF) [inline AIML/lib/aiml/_fakenameof.aiml]: Perhaps change "When hungry" to "When you're hungry"?

File: AIML/lib/aiml/_fakenameof.aiml [hunk 4]
```diff
@@ -309,6 +308,48 @@
...
+<li>A good way to keep healthy is to eat more Chinese food.</li>
```
Comment (FakeNameOF) [inline AIML/lib/aiml/_fakenameof.aiml]: Change "to keep healthy" to "to keep yourself hungry"

File: AIML/lib/aiml/_fakenameof.aiml [hunk 5]
```diff
@@ -309,6 +308,48 @@
...
+<li>All your fingers can't be of the same length.</li>
+<li>Everything happens for a reason.</li>
+<li>To be old and wise, you must first be young and stupid.</li>
+<li>Never ask what hot dogs are made of.</li>
+<li>Patience is a virtue, unless its against a brick wall.</li>
```
Comment (FakeNameOF) [inline AIML/lib/aiml/_fakenameof.aiml]: Change "its" to "it's"

File: AIML/lib/aiml/_fakenameof.aiml [hunk 6]
```diff
@@ -309,6 +308,48 @@
...
+<li>If you are afraid to shake the dice, you will never throw a six.</li>
+<li>You try hard, never to fail. You don't, never to win.</li>
+<li>It never pays to kick a skunk.</li>
+<li>If you are never patient, you will never get anything done. If you believe you can do it, you will be rewarded with success.</li>
+<li>Your smile brings happiness to everyone you meet.</li>
+<li>!nsfw is not for the faint of heart.</li>
+<li>ヽ༼ຈ͜لຈ༽ﾉ raise your dongers ヽ༼ຈ͜لຈ༽ﾉ</li>
+<li>It's about time you asked that special someone on a date.</li>
+<li>Put your unhappiness aside. Life is beautiful, be happy.</li>
+<li>If you never expect anything you can never be disappointed.</li>
```
Comment (FakeNameOF) [inline AIML/lib/aiml/_fakenameof.aiml]: Add a comma between anything and you (If you never expect anything, you can never be disappointed.)

File: AIML/lib/aiml/_fakenameof.aiml [hunk 7]
```diff
@@ -309,6 +308,48 @@
...
+<li>Follow your bliss and the Universe will open doors where there were once only walls. :blissful_creep:</li>
+<li>All the water in the world can't sink a ship unless it gets inside.</li>
+<li>Do what is right, not what you should.</li>
+<li>Bread today is better than cake tomorrow.</li>
+<li>It's all right to have butterflies fin your stomach. Just get them to fly in formation.</li>
```
Comment (FakeNameOF) [inline AIML/lib/aiml/_fakenameof.aiml]: Change to "butterflies in your stomach."

**Comment by FakeNameOF** [comment]
Approving changes.

**Pull request merged**